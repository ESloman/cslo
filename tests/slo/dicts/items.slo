var alpha = {"a": 1, "b": 2, "c": 3};
for (var pair in alpha.items()) {
    println("key: ", pair[0], ", value: ", pair[1]);
}
