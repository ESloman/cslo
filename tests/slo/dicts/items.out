key: c, value: 3
key: a, value: 1
key: b, value: 2
